modules/cxf/src/test/java/org/mule/runtime/module/cxf/ClientSimpleFrontendTestCase.java

```diff
@@ -9,19 +9,26 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
-import org.mule.functional.junit4.FunctionalTestCase;
+import org.mule.extension.http.internal.HttpConnector;
+import org.mule.extension.socket.api.SocketsExtension;
+import org.mule.functional.junit4.ExtensionFunctionalTestCase;
 import org.mule.runtime.core.api.message.InternalMessage;
 import org.mule.tck.junit4.rule.DynamicPort;
 
 import org.junit.Rule;
 import org.junit.Test;
 
-public class ClientSimpleFrontendTestCase extends FunctionalTestCase {
+public class ClientSimpleFrontendTestCase extends ExtensionFunctionalTestCase {
```

Comment:
Would it make sense to have an AbstractCxfTestCase with the annotated extensions stuff so as not to rewrite it everywhere?